package.json

```diff
@@ -13,11 +13,11 @@
     "coverage": "solidity-coverage",
     "networks-extract": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js",
     "networks-inject": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
-    "networks-reset": "mkdir -p build/contracts && npx truffle networks --clean && npm run networks-inject",
+    "networks-reset": "mkdir -p build/contracts && truffle networks --clean && npm run networks-inject",
```

Comment:
Why this change? It will require a global truffle

Reply:
No, npm scripts are run with `node_modules/.bin/` in the path automatically added.

Reply:
No, it will use the truffle specified in the dependencies.

I could not find a better source than this one:
https://dev.to/nebrius/comment/a9l9

But Nick said this in the past as well :)

Reply:
Here :) https://docs.npmjs.com/cli/run-script

Reply:
Or here now:
> [script] can also be any locally installed executable that is inside node_modules/.bin/
https://yarnpkg.com/lang/en/docs/cli/run/